ecc/README.md

```diff
@@ -8,56 +8,55 @@ Before your continue here make sure you have build ``ecc_enclave`` before.
 We refer to [ecc_enclave/README.md](../ecc_enclave). Otherwise, the build
 might fail with the message `ecc_enclave build does not exist!`.
 
-This is a go chaincode that is used to invoke the enclave. The chaincode logic
-is implemented in C++ as enclave code and is loaded by by the go chaincode as
-C library (``ecc/enclave/lib/enclave.signed.so``).  For more details on the
+This is a go chaincode that is used to invoke the enclave (and hence
+FPC chaincode). The chaincode logic is implemented in C++ as enclave
+code and is loaded by by the go chaincode as C library
+(``ecc/enclave/lib/enclave.signed.so``).  For more details on the 
 chaincode implementation see [ecc_enclave](../ecc_enclave).
 
+The enclave registry can be run in two modes, as a normal chaincode
```

Comment:
```suggestion
The FPC chaincode can be run in two modes, as a normal chaincode
```

Reply:
ah, missed that one in the almost-copy/paste from ercc text ..